src/parser/mod.rs

```diff
@@ -88,29 +88,59 @@ fn test_parse_value_data_2() -> Result<(), Error> {
     Ok(())
 }
 
+/// Parse a scss file by path.
+///
+/// Returns a vec of the top level items of the file (or an error message).
+pub fn parse_scss_path(path: &Path) -> Result<Vec<Item>, Error> {
+    do_parse_scss_path(path, &SourceName::root(path.display()))
+}
+
+pub fn parse_imported_scss_path(
+    path: &Path,
+    from: SourcePos,
+) -> Result<Vec<Item>, Error> {
+    let source = SourceName::imported(path.display(), from);
+    do_parse_scss_path(path, &source)
+}
+
+fn do_parse_scss_path(
```

Comment:
... then I think this function is only used in `parse_scss_path`.  If so, it should be inlined there.